src/compiler/crystal/semantic/method_missing.cr

```diff
@@ -38,7 +38,7 @@ module Crystal
     def define_method_from_method_missing(method_missing, signature, original_call)
       name_node = StringLiteral.new(signature.name)
       args_nodes = [] of ASTNode
-      named_args_nodes = nil
+      named_args_nodes = [] of NamedArgument
```

Comment:
@asterite I've reverted to  `named_args_nodes = nil`. The other changes remained.

:eyes: ?